MWWebPlugin/MWWebPlugin/WebStep/MWWebStep.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  MWWebStep.swift
+//  MWWebPlugin
+//
+//  Created by Xavi Moll on 23/12/20.
+//
+
+import Foundation
+import MobileWorkflowCore
+
+public class MWWebStep: ORKStep {
+    
+    let url: URL
+    
+    init(identifier: String, url: URL) {
+        self.url = url
+        super.init(identifier: identifier)
+    }
+    
+    required init(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    public override func stepViewControllerClass() -> AnyClass {
+        return MWWebViewController.self
+    }
+}
+
+extension MWWebStep: MobileWorkflowStep {
+    public static func build(step stepInfo: StepInfo, services: MobileWorkflowServices) throws -> ORKStep {
+        if let urlString = stepInfo.data.content["url"] as? String, let url = URL(string: urlString) {
+            return MWWebStep(identifier: stepInfo.data.identifier, url: url)
+        } else {
+            throw NSError(domain: "io.mobileworkflow.web", code: 0, userInfo: [NSLocalizedDescriptionKey:"URL missing from the JSON"])
```

Comment:
Minor comment: For better maintenance, it is better to avoid this ID hardcoded here and in the plugin. To have a complete association of the plugin ID with this error domain, it is better if both access the same constaint